create_box/main.py

```diff
@@ -1,5 +1,13 @@
 def create_box(height, width, character):
-    pass
+    height = int(height)
+    width = int(width)
+    text = ''
+    if height >= 1 and width >=1:
+        for i in range(0, height):
```

Comment:
You can omit the 0 in range(start, stop[, step]) and just pass in the height. When the start value is omitted the default value is 0.